### PR TITLE
Docker: Add support for CLI and UI modesFt/user interface

### PR DESCRIPTION
**Pull request by shreyaprabhu-g**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -68,13 +68,23 @@ python3 DAGify.py --help
...
-docker run -it --env-file=.env.example -v $(pwd):/app localhost/dagify:source
+docker run -p 8000:8000 -it --env-file=.env.example -v $(pwd):/app -e RUN_MODE=cli localhost/dagify:source;
+
+```
+
+#### With User Interface
+```bash
+
+docker build -t localhost/dagify:source . 
+docker run -p 8000:8000 -it --env-file=.env.example -v $(pwd):/app -e RUN_MODE=ui localhost/dagify:source
```
Comment (KonradSchieban) [inline README.md:87]: does the Docker run with UI actually need the env file and volume mount as parameters?

**Review by KonradSchieban — CHANGES_REQUESTED** [review]
Minor comment on the Readme

File: README.md (hunk 1 above)
Comment (timbohiatt) [reply README.md:87]: I expect the underlying code is still expecting the mount dow writing the files that would then be served up for downloading.

Comment (timbohiatt) [reply README.md:87]: just cause of the way it's using the existing code.

Comment (shreyaprabhu-g) [reply README.md:87]: Yes, the UI does not need env file or volume mount.

**Review by KonradSchieban — APPROVED** [review]

**Pull request merged**